csync/src/csync_owncloud.c

```diff
@@ -826,12 +856,28 @@ int owncloud_set_property(CSYNC* ctx, const char *key, void *data) {
     if( c_streq(key, "redirect_callback")) {
         if (data) {
             csync_owncloud_redirect_callback_t* cb_wrapper = data;
-
             ctx->owncloud_context->dav_session.redir_callback = *cb_wrapper;
         } else {
             ctx->owncloud_context->dav_session.redir_callback = NULL;
         }
     }
+    if( c_streq(key, "SSLClientCerts")) {
+        if(ctx->clientCerts != NULL) {
```

Comment:
i don't understand where i shoud put that cleanup code.

i tried to put it at the very end of this function but then the client receives a 'ssl handshacke error'

please provide details.